docker.go

```diff
@@ -138,20 +137,28 @@ func (provider *DockerProvider) loadDockerConfig(dockerClient *docker.Client) *C
 			log.Debugf("Filtering disabled container %s", container.Name)
 			return false
 		}
+		_, errRule := provider.getLabel(container, "traefik.frontend.rule")
+		_, errValue := provider.getLabel(container, "traefik.frontend.value")
+
+		if errRule != nil && errValue == nil || errRule == nil && errValue != nil {
```

Comment:
Looks weird :sweat_smile: 
What about a method that does that, something `getExclusiveLabel` or something — that would return an error if both or none are set :

```go
if _, err := provider.getExclusiveLabel(container, []string{"traefik.frontend.rule", "traefik.frontend.value"}) {
    log.Debugf("…")
    return false
}
```

As you don't use the return value, it could even be a `validateSthg` function that do not return anything else than the answer but…